www/styles/Shared.module.css

```diff
@@ -55,13 +63,13 @@
 
 .page::-webkit-scrollbar-corner {
   background: var(--background);
-}
+} */
 
 /* thumb refers to the bar on the slider */
-.page::-webkit-scrollbar-thumb {
-  background-image: linear-gradient(var(--agenda), var(--agendaThree));
+/* .page::-webkit-scrollbar-thumb {
+  background-image: linear-gradient(var(--), var(--primary));
```

Comment:
Are we not doing the scroll bar?

Reply:
Yeah we can, we just need to re-adjust it. It was causing some issues with the styles that I didn't want to fix immediately.